doc/notebooks/grass_jupyter.ipynb

```diff
@@ -43,7 +43,7 @@
     "session = gj.init(\"../../data/grassdata\", \"nc_basic_spm_grass7\", \"user1\")\n",
     "\n",
     "# Set computational region to the elevation raster.\n",
-    "gs.run_command(\"g.region\", raster=\"elevation\")"
+    "gs.parse_command(\"g.region\", raster=\"elevation\")"
    ]
   },
```

Comment:
It's nice that it prevents it from printing to the shell but it also prints an empty dictionary as the cell output. Perhaps we could add the print flag so at least the dictionary is full?

Reply:
Done in 396de52283dc39053831bed48fee0be94477b593

Reply:
![Screenshot from 2022-04-15 19-30-16](https://user-images.githubusercontent.com/5449060/163652390-c67dc7f9-b4cd-4fc1-a06d-51fbc40fcf32.png)

Reply:
Note that the notebooks are not tested in the CI. Only manually locally or in Binder.

It is possible to do that and also to generate HTMLs from them. We need to test them, that's for sure, although propagating errors is challenging, so manual check in Binder may still be needed.

Maybe generating the HTMLs is needed too. ...If done also on a release server, maybe we can have better links for the documentation. ...Totally unrelated to this PR.

Reply:
> Note that the notebooks are not tested in the CI. Only manually locally or in Binder.

Ops, syntax fixed.